ADOPTERS.md

```diff
@@ -14,6 +14,33 @@ This is a meta user; please feel free to document specific OpenShift users!
 
 All OpenShift clusters use the Prometheus Operator to manage the cluster monitoring stack as well as user workload monitoring. This means the Prometheus Operator's users include all OpenShift customers.
 
+## Giant Swarm
```

Comment:
All OpenShift Users might be a bit misleading if this sorted? :D Maybe we can rename All OpenShift users -> OpenShift users and sort it :)

Reply:
I agree that putting the product name / company first makes sense here

Reply:
Agree too, what about s/All OpenShift Users/OpenShift/

Reply:
Fixed :)